packages/editor/src/components/post-card-panel/index.js

```diff
@@ -98,20 +111,14 @@ export default function PostCardPanel( { className, actions } ) {
 					{ actions }
 				</HStack>
 				<VStack className="editor-post-card-panel__content">
-					{ ( description ||
-						lastEditedText ||
-						showPostContentInfo ) && (
-						<VStack
-							className="editor-post-card-panel__description"
-							spacing={ 2 }
-						>
-							{ description && <Text>{ description }</Text> }
-							{ showPostContentInfo && <PostContentInfo /> }
-							{ lastEditedText && (
-								<Text>{ lastEditedText }</Text>
-							) }
-						</VStack>
-					) }
+					<VStack
+						className="editor-post-card-panel__description"
+						spacing={ 2 }
+					>
+						{ showPostExcerptPanel && <PrivatePostExcerptPanel /> }
+						{ showPostContentInfo && <PostContentInfo /> }
+						{ lastEditedText && <Text>{ lastEditedText }</Text> }
+					</VStack>
```

Comment:
Where is this rendered so I can test it? If I'm not wrong it's the top of the inspector no? I didn't see the excerpt there in my testing, so I'm missing something.

Reply:
It's in the top of the inspector yes, but for patterns, templates and template parts.

Then it will be shown if there is a description/excerpt and if it's a user create entity it will be editable too..